LSPS2/README.md

```diff
@@ -772,8 +772,7 @@ where the next hop is `jit_channel_scid`:
   step:
   * The current time is strictly less than
     `opening_fee_params.valid_until`.
-  * All received payment parts sum up to at least `payment_size_msat`
-    plus `opening_fee`.
+  * All received payment parts sum up to at least `payment_size_msat`.
```

Comment:
Is this effectively saying that `opening_fee` is not charged? Or is it subtractive?